packages/makecode-core/src/downloader.ts

```diff
@@ -139,14 +153,26 @@ export async function downloadAsync(
             try {
                 log("Checking for updates (only happens once daily)...")
                 needsUpdate = await hasNewManifestAsync()
+                if (!needsUpdate) {
+                    // fetch new target config as that is 'live'
+                    const targetConfig = await fetchTargetConfig(
+                        info.webConfig.cdnUrl,
+                        info.webConfig.targetId,
+                        info.targetVersion
+                    );
+                    if (targetConfig) {
+                        info.targetConfig = targetConfig;
+                        await saveInfoAsync();
+                    }
+                }
             } catch (e) {
                 log(
                     `Error checking for updates; will try again tomorrow (use -u flag to force); ${e.message}`
                 )
             }
         }
         if (!needsUpdate) return loadFromCacheAsync()
-    } else {
+    } else if (useCached) {
```

Comment:
Whoops, didn't mean to sneak this in in particular as we don't hit it, but it felt weird looking at it that it'd still use cache when `forceUpdate == true` in `loadEditorAsync` Can cut though